server-api/src/functional-api/notifications/callouts/post-comments.it-spec.ts

```
@@ -30,12 +33,12 @@ let postCommentsIdSpace = '';
 let postCommentsIdSubspace = '';
 let postCommentsIdSubsubspace = '';
 let messageId = '';
-let preferencesPostConfig: any[] = [];
-let preferencesPostCommentsConfig: any[] = [];
+export let preferencesPostCreatedConfig: any[] = [];
+export let preferencesPostCommentsCreatedConfig: any[] = [];
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)